.github/workflows/feature-branch-deployment.yaml

```diff
@@ -2,9 +2,9 @@ name: Feature Branch Deployment
 
 on:
   workflow_dispatch:
-  push:
-    branches:
-      - '[0-9]+-**'
+#   push:
+#     branches:
+#       - '[0-9]+-**'
```

Comment:
This should be uncommented before merging

Reply:
It was for testing. Done.